src/tools/pub/cmd/_pub-outdated-output.html

```diff
@@ -13,7 +13,7 @@
 transitive dev_dependencies: all up-to-date
 
 1 upgradable dependency is locked (in pubspec.lock) to an older version.
-To update it, use `pub upgrade`.
+To update it, use `dart pub upgrade`.
```

Comment:
Unfortunately, the output still says `pub upgrade`, not `dart pub upgrade` (at least in 2.10.4 stable, and we document the stable version). I've filed https://github.com/dart-lang/sdk/issues/44612.

/cc @mit-mit

Reply:
Fixing in https://github.com/dart-lang/pub/pull/2823 (note: after that is merged, it'll take a bit of time to roll the change in the the SDK. But I think I'd be fine to update docs as soon as the PR is merged).